addons/account/views/report_invoice.xml

```diff
@@ -23,7 +23,10 @@
 
                     <div id="informations" class="row mt32 mb32">
                         <div class="col-auto mw-100 mb-2" t-if="o.invoice_date" name="invoice_date">
-                            <strong>Invoice Date:</strong>
+                            <span t-if="o.type == 'out_invoice'"><strong>Invoice Date:</strong></span>
+                            <span t-if="o.type == 'out_refund'"><strong>Credit Note Date:</strong></span>
+                            <span t-if="o.type == 'in_refund'"><strong>Vendor Credit Note Date:</strong></span>
```

Comment:
When are you printing vendor documents?
You should receive then and never generate them

Reply:
In fact my needs is about out_refund, I wanted to make sure all case to be covered, do you agree an else case with `Date` label ?